tools/api-tests/yarn.lock

```diff
@@ -65,41 +65,41 @@
     imurmurhash "^0.1.4"
```

Comment:
this file appears not to be in use? https://github.com/elifesciences/elife-xpub/pull/2350